weblogic-azure-aks/src/main/arm/scripts/appgw-helm-config.yaml.template

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+#
+# Based on https://raw.githubusercontent.com/Azure/application-gateway-kubernetes-ingress/master/docs/examples/sample-helm-config.yaml
+
+# This file contains the essential configs for the ingress controller helm chart
+
+# Verbosity level of the App Gateway Ingress Controller
+verbosityLevel: 3
+
+################################################################################
+# Specify which application gateway the ingress controller will manage
+#
+appgw:
+    subscriptionId: @SUB_ID@
+    resourceGroup: @APPGW_RG_NAME@
+    name: @APPGW_NAME@
+    usePrivateIP: false
+
+    # Setting appgw.shared to "true" will create an AzureIngressProhibitedTarget CRD.
+    # This prohibits AGIC from applying config for any host/path.
+    # Use "kubectl get AzureIngressProhibitedTargets" to view and change this.
+    shared: false
+
+################################################################################
+# Specify which kubernetes namespace the ingress controller will watch
```

Comment:
kubernetes -> Kubernetes

Reply:
Fixed in https://github.com/galiacheng/weblogic-azure/commit/37f4036c9bd1b88938c9697d764be5b303a6378b